### ISSUE TITLE
Integration with Pardot

### ISSUE THREAD

**Issue by schaferda** [zaus/forms-3rdparty-integration#91]
Has anyone else seen the situation where a single POST to Pardot has triggered the Form Handler twice?

**Comment by Just-Johnny** [comment]
Yes, in some cases it seems to be sending several. Sometimes 3, 8, 13... etc.
